desktop/src/com/mygdx/game/client/network/ComponentMessageListener.java

```diff
@@ -37,6 +44,19 @@ public ComponentMessageListener(
       }
       return true;
     });
+    registerHandler(Position.class, (webSocket, packet) -> {
+      System.out.println("Read position component");
+      var entity = networkEntityManager.getWorldEntity(packet.getEntityId());
+      Position newPosition = (Position) packet.getComponent();
+
+      Position unitPosition = positionMapper.get(entity);
+      unitPosition.setPosition(newPosition.getPosition());
+      if (entity == -1) {
```

Comment:
To lepiej sprawdzać na samym początku, bo to robimy komponenty na entity = -1, a to ma być taki NO_ENTITY taka wartość, która jest błędem.

Najfajniej byłoby jakby byłoby tworzone nowe entity na podstawie danych z komponentu, który został wysłany i po prostu zostałoby to wprowadzone do world/engine i obsługiwane przez te systemy, które są w stanie pracować na mniejszej ilości danych.